src/_pytest/runner.py

```diff
@@ -49,6 +50,8 @@ def pytest_terminal_summary(terminalreporter):
         dlist = dlist[:durations]
 
     for rep in dlist:
+        if verbose < 2 and rep.duration < 0.01:
```

Comment:
should this take into account rounding (`< 0.005` or whatever?)

is `0.009` displayed as 0.01? (I don't know, this could be fine already!)

Reply:
It uses `02.2f` for formatting:

```
>>> '%02.2fs' % 0.009
'0.01s'
```

So I think this is fine as is. 👍

Reply:
wouldn't that mean it isn't fine since it's not displayed as 0.0s?

Reply:
Oh duh you are right, sorry.

Reply:
it's certainly an edge case of an edge case though  :laughing: